### DISCUSSION TITLE
Dynamic routes

### DISCUSSION THREAD

**Discussion by silviucs** [vikejs/vike#1294] [Discussion]
Hello all,

I need to build a web app and now I'm in the "research" phase - I'm looking for the best framework to fulfil my requirements. So far, Vike seems the best, but I have a blocker. I need to be able to have URLs like this: **/en/about** and **/pt/sobre** (basically change the url based on the language). 

Is it possible (if yes, how) to do this with Vike?

Cheers

**Comment by brillout** [comment]
You should be able to do that. Have you seen https://vike.dev/i18n? It requires more boilerplate but it gives you a lot more control than what other frameworks usually provide. There is also https://github.com/vikejs/vike/issues/1000 but it won't add any functionality: it's only about implementing a trivial DX for simple i18n use cases.

**Reply by silviucs** [reply]
Thank you for your quick reply - I'll have a look at i18n.
